packages/wonder-blocks-modal/src/components/flexible-panel.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import {PropsFor, View} from "@khanacademy/wonder-blocks-core";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+
+import {focusStyles} from "@khanacademy/wonder-blocks-styles";
+import {StyleSheet} from "aphrodite";
+import {semanticColor} from "@khanacademy/wonder-blocks-tokens";
+import ModalContent from "./modal-content";
+import CloseButton from "./close-button";
+import theme from "../theme";
+
+type RenderProps = {
+    title: React.ReactNode | string;
+};
+
+type Props = {
+    /**
+     * The main heading of the FlexiblePanel. Used to label the dialog.
+     */
+    title?: React.ReactNode | string;
```

Comment:
Interestingly, this one needed to stay as `React.ReactNode` to work with the renderedTitle from `FlexibleDialog`.